enaml/applib/live_editor_model.py

```diff
@@ -232,6 +238,38 @@ def refresh_view(self):
         else:
             self.traceback = ''
 
+    def autocomplete(self, source, position):
+        """ Obtain autocompletion suggestions for the source text 
+        using jedi if available.
+        
+        """
+        if not COMPLETION_AVAILABLE or not source:
+            return []
+        try:
+            #: Use jedi to get suggestions
```

Comment:
Please use `#:` for documenting attributes but no for general comments. The point in using `#:` for attribute is because Sphinx can pick those up when generating documentation.

Reply:
Thanks. I just got into the habit of doing it that way, will change it.